### ISSUE TITLE
Update roles and access profiles in bulk

### ISSUE THREAD

**Issue by yannick-beot-sp** [yannick-beot-sp/vscode-sailpoint-identitynow#84]
**Is your feature request related to a problem? Please describe.**
At this moment, it is only possible to create roles or access profiles in bulk and not update them.

**Describe the solution you'd like**
It should be possible to submit a CSV to update roles or access profiles

**Describe alternatives you've considered**
Scripts (leveraging SailPoint SDK), Role Importer (PS tool), etc.

**Additional context**
Things to be considered/defined:
- Should it be "create or update" type of import, or should we have 2 different commands? One for import, One for Update? As of now, there is an icon in the menu for import. Probably the expected behavior is to "create or update"
- In the context of an update, not all fields are required. For the instance, the owner is already set and shall not change
- JSON Patching should be considered

**Comment by fernando-delosrios-sp** [comment]
Your current implementation is fantastic from my point of view, Yannick. As an improvement I would allow for updates/overwrites when a given role/AP is already present. I don't think you need to consider partial inputs or split imports and updates, since anyone can use your export output and modify existing roles or add new ones without much effort.

Congrats. Excellent work as always.

**Comment by Jaewon-Shin-KR** [comment]
@fernando-delosrios-sp 
I agree with your opinion. It would be really beneficial for users if Roles/APs with the same name could be updated instead of creating duplicates. Additionally, I think it’s a good idea to explicitly separate the functionalities into bulk update and bulk creation to avoid potential user errors where similar objects might be created instead of updating existing ones.
Supporting bulk updates in any form would be a fantastic improvement.

**Comment by chadcarlton** [comment]
Export, modify, import would be the desired functionality.

This would be amazing in the VSCode extension!

**Comment by yannick-beot-sp** [comment]
@chadcarlton export and import of roles and access profiles are already possible in the extension. Check the menu for Roles and Access Profiles in your tenant in the extension.
The question is more: how to handle the modify?
Is it a separate command or should I manage a "add or update" operation?
Modification is not always desired. "Add or update" operation are quite expensive as it needs to check before hand if the role or access profile exists, etc.
